addons/event_sale/models/event.py

```diff
@@ -21,6 +21,7 @@ class event_event(models.Model):
     badge_back = fields.Html('Badge Back', translate=True, states={'done': [('readonly', True)]})
     badge_innerleft = fields.Html('Badge Innner Left', translate=True, states={'done': [('readonly', True)]})
     badge_innerright = fields.Html('Badge Inner Right', translate=True, states={'done': [('readonly', True)]})
+    is_unlimited = fields.Boolean('Unlimited Tickets', compute='_compute_seats_availability')
```

Comment:
hello @tde-banana-odoo , your example is not possible due to calculation of the available seats(seat_max) of event is based on total no of available seats(seat_max) of ticket. so after event_Sale seat_Available are calculated based on tickets.please can you give your opinion? Thanks